mava/utils/jax_training_utils.py

```diff
@@ -58,13 +98,24 @@ def compute_running_mean_var_count(
     delta = batch_mean - mean
     tot_count = count + batch_count
 
-    new_mean = mean + delta * batch_count / tot_count
+    mean = mean + delta * batch_count / tot_count
     m_a = var * count
     m_b = batch_var * batch_count
     M2 = m_a + m_b + jnp.square(delta) * count * batch_count / tot_count
-    new_var = M2 / tot_count
+    var = M2 / tot_count
+    std = jnp.sqrt(var)
     new_count = tot_count
-    new_std = jnp.sqrt(new_var)
+
+    # This assumes the all the features we don't want to
+    # normalise are all always at the front.
+    new_mean = jnp.zeros_like(mean)
+    new_var = jnp.zeros_like(var)
+    new_std = jnp.ones_like(std)
+
+    indices = np.r_[axes]
```

Comment:
I think if you just passed in axes as an `np.ndarray` you would be able to avoid this `np.r_` call and should simplify this whole process substantially